curvefs/src/tools/create/curvefs_create_fs.cpp

```diff
@@ -121,6 +131,20 @@ void CreateFsTool::AddUpdateFlags() {
     AddUpdateFlagsFunc(curvefs::tools::SetEnableSumInDir);
 }
 
+namespace {
+google::protobuf::RepeatedPtrField<::std::string> ParseVolumeCluster(
```

Comment:
If so, it will fail at fetching volume size and segment size from bs cluster when creating fs. 
The reason I don't verify hosts is IP and ports can be valid but to a non-exist cluster.